@here/generator-harp.gl/generators/app/index.js

```diff
@@ -27,16 +27,40 @@ module.exports = class extends Generator {
 
     writing() {
         this.fs.copyTpl(
-            [
-                this.templatePath('*.*'),
-                this.templatePath('scripts/*.*')
-            ],
-            this.destinationPath(''),
+            [this.templatePath("*.*"), this.templatePath("scripts/*.*")],
+            this.destinationPath(""),
             {
                 packagename: this.answers.packagename,
                 access_token: this.answers.access_token,
                 generator_version: version
             }
         );
     }
+
+    install() {
+        // npmInstall conflicts with running install-peerdeps, use spawnCommandSync instead
+        this.spawnCommandSync("npx", ["npm", "install", "--no-save", "install-peerdeps"]);
```

Comment:
why do we need to call `npx npm install ...`. Is the npx necessary?

Reply:
Nope, good point thanks